### DISCUSSION TITLE
How to use the osmand AidlApi to put routing memory default higher

### DISCUSSION THREAD

**Discussion by christianCasRo** [osmandapp/OsmAnd#16550] [Q&A]
I use osmand aidlapi, but I don´t know how to implement the option of the plugin development option "Routing memory" to put default more RAM memory to my osmand, I haven’t found the way (sorry). I´m trying to use PluginParams and I don´t know if the correct way is PluginParams then "pluginId".
This is because when I use osmand with a 150km route is very slow and don’t use more than 256 mb of my 8GB.
I already change from osmand options, but I would like to have a way or profile (I try change profile options and then save but I can't save that option particularly)

Thanks you in advance 💟

**Comment by Sonwon1** [comment]
Maybe it is called disable safe mode?

**Reply by christianCasRo** [reply]
It's already disable by default
